docs/dev/table/tableApi.md

```diff
@@ -373,7 +373,7 @@ Table result = orders
     <tr>
       <td>
         <strong>Distinct</strong><br>
-        <span class="label label-primary">Batch</span>
+        <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span>
```

Comment:
I think, Distinct always produces updates no matter the input table produces updates. Because the distinct is translated into GroupBy.